### ISSUE TITLE
Add background confetti image

### ISSUE THREAD

**Issue by Feissit** [CUCentralAdvancement/essential-cu#341]
High priority

Add the transparent confetti image to the top of the story page

Background image link: https://www.dropbox.com/s/ut8572o4cok12nn/Web-bckgd-01.png?dl=0

CSS:
.web-bckgd-01-copy-2 { 
height: 804.94px; 
width: 1440px; 
}

<img width="795" alt="Screen Shot 2021-10-13 at 12 49 09 PM" src="https://user-images.githubusercontent.com/91230278/137194995-5301a2d7-aed2-46ca-8a3f-23d571c54130.png">

**Comment by alexfinnarn** [comment]
@duquennoisa @imkleats I fiddled around with this but couldn't seem to get the border to go away. The gradient lightens the line. Oddly, adding certain padding + absolute positions made the border/line disappear on my screen. I'm not sure if browsers will render this different as well...I tried cropping the image to get the border off but did not work for me...last year's image with the same CSS and no line...

One thought might be to export with a gradient on it. I did notice a squiggly thing right at the bottom of the pattern so it makes sense to fade that out, I think.
